docs/BuildFromSource.md

```diff
@@ -58,11 +53,40 @@ git submodule update --init --recursive
 
 ## Building in Visual Studio / Code
 
-Before opening our .sln files in Visual Studio or VS Code, executing the following on command-line:
-```
-.\build.cmd /t:Restore
-```
-This will download required tools.
+Before opening our .sln files in Visual Studio or VS Code, you need to perform the following actions.
+
+1. Executing the following on command-line:
+   ```
+   .\build.cmd /p:SkipTests=true
```

Comment:
If I drop `/p:_ProjectsOnly=true` I get 4.7.2 errors in razor:
```
C:\Users\chrross\.dotnet\x64\sdk\3.0.100-preview-009750\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.7.2" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [D:\github\AspNetCore\modules\Razor\src\Microsoft.CodeAnalysis.Remote.Razor\Microsoft.CodeAnalysis.Remote.Razor.csproj]
C:\Users\chrross\.dotnet\x64\sdk\3.0.100-preview-009750\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.7.2" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [D:\github\AspNetCore\modules\Razor\src\Microsoft.VisualStudio.Editor.Razor\Microsoft.VisualStudio.Editor.Razor.csproj]
C:\Users\chrross\.dotnet\x64\sdk\3.0.100-preview-009750\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.7.2" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [D:\github\AspNetCore\modules\Razor\src\Microsoft.VisualStudio.Mac.LanguageServices.Razor\Microsoft.VisualStudio.Mac.LanguageServices.Razor.csproj]
C:\Users\chrross\.dotnet\x64\sdk\3.0.100-preview-009750\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.7.2" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [D:\github\AspNetCore\modules\Razor\src\Microsoft.VisualStudio.LanguageServices.Razor\Microsoft.VisualStudio.LanguageServices.Razor.csproj]
```

Reply:
Hmm. I'll put /p:_ProjectsOnly=true back in the instructions for now. I thought we had a PackageRef to netfx 4.7.2, but apparently that doesn't work...